src/schemas/checkoutput.py

```diff
@@ -0,0 +1,16 @@
+from jsl import Document, DocumentField
+from jsl.fields import ArrayField, StringField
+from snactor.registry.schemas import registered_schema
+
+
+@registered_schema('1.0')
+class CheckEntry(Document):
+    check_id = StringField()
+    status = StringField()
```

Comment:
AFAICT this is an `enum` value, can you also list the possible options here? ('success' and 'error' i guess)